### PR TITLE
Remove MA services and Refactor BO and SI PS services

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: Makefile_services.mk [hunk 1]
```diff
@@ -578,96 +494,40 @@ service-ioc-si-ps-fams-stop:
...
+		--extra-vars "services_to_run_fams='' services_to_run_trims='' ctrl_service_state=started" \
+		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps.yml
 
 service-ioc-si-ps-trims-stop:
 	@echo "Stopping SI PS Trims IOC services"
 	@echo "Please enter password for sirius user."
 	ansible-playbook \
-		--extra-vars "ctrl_service_state=stopped" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ps-trims.yml
+		--extra-vars "services_to_run_corrs='' services_to_run_trims='' ctrl_service_state=stopped" \
```
Comment (anacso17) [inline Makefile_services.mk]: shouldn't it be `--extra-vars "services_to_run_corrs='' services_to_run_fams='' ctrl_service_state=stopped" \` ?

File: Makefile_services.mk [hunk 2]
```diff
@@ -578,96 +494,40 @@ service-ioc-si-ps-fams-stop:
...
-		--extra-vars "ctrl_service_state=stopped" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ma-trims.yml
-
-service-ioc-si-ma-trims-start:
-	@echo "Starting SI MA Trims IOC services"
-	@echo "Please enter password for sirius user."
-	ansible-playbook \
-		--extra-vars "ctrl_service_state=started" \
-		-u sirius -k --ask-become-pass playbooks-services/playbook-service-ioc-si-ma-trims.yml
+		--extra-vars "services_to_run_corrs='' services_to_run_trims='' ctrl_service_state=started" \
```
Comment (anacso17) [inline Makefile_services.mk]: same here

File: Makefile_services.mk (hunk 1 above)
Comment (xresende) [reply Makefile_services.mk]: yes! thanks. fixed.

File: Makefile_services.mk (hunk 2 above)
Comment (xresende) [reply Makefile_services.mk]: fixed.

**Pull request merged**